### DISCUSSION TITLE
https://vuepress-theme-hope.github.io/v2/zh/guide/interface/theme-color.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#577] [Announcements]
# https://vuepress-theme-hope.github.io/v2/zh/guide/interface/theme-color.html

主题允许你自定义主题颜色，甚至提供选择器。

https://vuepress-theme-hope.github.io/v2/zh/guide/interface/theme-color.html

**Comment by heifree** [comment]
dvf
